contributing/development/code_style_guidelines.rst

```diff
@@ -292,15 +302,29 @@ command:
 - The path can point to several files, either one after the other or using
   wildcards like in a typical Unix shell.
 
+
 Pre-commit hook
-^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~
+
+For ease of use, we provide hooks for Git with the `pre-commit <https://pre-commit.com/>`__
+Python framework that will run black automatically on all your commits with the 
+correct version of black.
```

Comment:
```suggestion
Python framework that will run ``black`` automatically on all your commits with the 
correct version of ``black``.
```

Not sure if this is best, but we should probably format "black" *in some way*, as otherwise it can be confusing, as it is just another normal english word, all lowercase.

Reply:
Okay! Then i will also format clang-format as `clang-format`

Reply:
actually - after doing this I realised that both clang-format and black are used unformatted a lot in the page. I think it would be best to be consistent. So do you think it would be better for all to be formatted, or all to be unformatted?

Reply:
Agree it should be consistent; lets see if someone else chimes in, don't want you to spend all that time on work others may disagree with.

Reply:
I have no strong opinions on formatting. However ``black`` should have an uppercase "B".

Reply:
More info: https://github.com/godotengine/godot/issues/88901 doesn't uppercase `black`, and `black` is used multiple times in the article without the uppercase - so its also an inconsistency. I'm not sure how its used throughout the rest of the Godot docs